doc/rados/operations/pools.rst

```diff
@@ -314,9 +314,10 @@ You may set values for the following keys:
 
 ``min_size``
 
-:Description: Sets the minimum number of replicas required for I/O.
-              See `Set the Number of Object Replicas`_ for further details.
-              Replicated pools only.
+:Description: Sets the minimum number of replicas or shards required for I/O.
+              For further details with **replicated pools**, see `Set the Number of Object Replicas`_
+              or `Erasure Coded Pool Recovery <../erasure-code/#erasure-coded-pool-recovery>`_
```

Comment:
Please do not use paths (relative or absolute are both problematic). The reasoning and recommended alternatives are described at: https://github.com/ceph/ceph/blob/master/doc/start/documenting-ceph.rst#toc-and-hyperlinks

Specifically:

```
Document authors should prefer to use the :ref: syntax where a link target contains a specific unique identifier (e.g., .. _unique-target-id:), and a reference to the target specifically references the target (e.g., :ref:`unique-target-id`) so that if source files are moved or the information architecture changes, the links will still work. 
```